test/one/profiler/test/Runner.java

```diff
@@ -221,14 +222,21 @@ public static void main(String[] args) throws Exception {
 
         List<RunnableTest> allTests = getRunnableTests(args);
         final int testCount = allTests.size();
+        boolean hasOnly = false;
```

Comment:
It's not very convenient to use and not very nice design-wise. I initially thought that in order to run selected tests, one would specify an argument with test names. This should already work: `make test TESTS=nativemem`. This mechanism can be further extended to allow execution of individual tests, not just test sets.

Reply:
Removing `only`.